<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Unit converter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/res/favicon.ico</normaloff>:/icon/res/favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="unitsGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>62</height>
        </size>
       </property>
       <property name="title">
        <string>Units</string>
       </property>
       <widget class="QComboBox" name="unitBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="inputgroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>1000</height>
        </size>
       </property>
       <property name="title">
        <string>Input</string>
       </property>
       <widget class="QLineEdit" name="inputLineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>191</width>
          <height>23</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="inputButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>21</width>
          <height>24</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Convert!</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/res/convert.png</normaloff>:/icon/res/convert.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="inputBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>181</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>31</width>
          <height>24</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Switch units!</string>
        </property>
        <property name="statusTip">
         <string>Tip: CTRL+S to switch ;)</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/res/invert.png</normaloff>:/icon/res/invert.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="shortcut">
         <string notr="true">Ctrl+S</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="convertlineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string notr="true">0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="convertButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>160</y>
          <width>21</width>
          <height>24</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Copy the result.</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/res/copy.png</normaloff>:/icon/res/copy.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="convertBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>181</width>
          <height>24</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>unitBox</tabstop>
  <tabstop>inputLineEdit</tabstop>
  <tabstop>inputBox</tabstop>
  <tabstop>convertBox</tabstop>
  <tabstop>convertButton</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>convertlineEdit</tabstop>
  <tabstop>inputButton</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>convertButton</sender>
   <signal>clicked()</signal>
   <receiver>convertlineEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>convertButton</sender>
   <signal>clicked()</signal>
   <receiver>convertlineEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
